client/betterblend/src/components/Track.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import '../css/playlist.css';
+import axios from 'axios';
+import 'font-awesome/css/font-awesome.min.css';
+
+class Track extends Component {
+
+	constructor() {
+		super();
+		this.state = {
+			classnames: '',
+			tracks: []
+		}
+	}
+
+	componentDidMount() {
+		this.getPlaylists();
+	}
+
+	getPlaylists = () => {
+		axios.get('/api')
+		.then(data => this.setState({tracks: data.data.tracks}));
+	};
+
+	render() {
+		return (
+			<div className={this.state.classnames}>
```

Comment:
Also wondered, why is it `<div className={this.state.classnames}>` and not `<div class={this.state.classnames}>`

Reply:
in react it is className not class, and each component has constructor and inside theres 'state' in which u can modify and put it inside html like `{this.state.<whatever>}` and you can setState as well.

Reply:
yaya i get the state component, i mean in the div object. why is it `className` and not `class`?

Reply:
it has to be `className` inside any tag in react, `class=` doesnt work
[stackoverflow link](https://stackoverflow.com/questions/46989454/class-vs-classname-in-react-16)